pkg/tenant/auth.go

```diff
@@ -450,21 +450,46 @@ func (s *Server) VerifyEmail(c *gin.Context) {
 		return
 	}
 
-	// Make the verify request to Quarterdeck
 	req := &qd.VerifyRequest{
 		Token: params.Token,
 	}
-	if err = s.quarterdeck.VerifyEmail(c.Request.Context(), req); err != nil {
+
+	// Parse the orgID if provided
+	if params.OrgID != "" {
```

Comment:
I'm thinking that the frontend should provide the `orgID` if it's available, otherwise invited users might be logged into the wrong organization.